src/displayapp/screens/WatchFaceDigital.cpp

```diff
@@ -46,6 +48,11 @@ WatchFaceDigital::WatchFaceDigital(DisplayApp* app,
   lv_label_set_text_static(bleIcon, Symbols::bluetooth);
   lv_obj_align(bleIcon, batteryPlug, LV_ALIGN_OUT_LEFT_MID, -5, 0);
 
+  alarmIcon = lv_label_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_text_color(alarmIcon, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(0x999999));
```

Comment:
I think the grey alarm icon doesn't feel quite right. I would expect it to be white
```suggestion
```

![WatchFaceDigital white alarm icon](https://user-images.githubusercontent.com/9076163/168667771-5d659712-f8fb-42fe-82e1-91781458e4df.png)